### PR TITLE
Adjust UploadToolbarAction to be consistent to other parts of the system

### PR DESCRIPTION
**Pull request by niklasnatter**
We use `snake_case` for the keys of the options of all other toolbar actions.

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/UploadToolbarAction.js [hunk 1]
```diff
@@ -131,20 +227,20 @@ export default class UploadToolbarAction extends AbstractListToolbarAction {
...
-            throw new Error('The "routeName" option must be a string!');
+            throw new Error('The "route_name" option must be a string!');
         }
 
         return symfonyRouting.generate(routeName, this.requestParameters);
     }
 
-    @computed get errorCodeMapping(): $ReadOnly<Object> {
-        const {errorCodeMapping = {}} = this.options;
+    @computed get errorCodeMessages(): $ReadOnly<Object> {
```
Comment (nnatter) [inline src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/UploadToolbarAction.js:239]: this makes the naming consistent with a similar option introduced in https://github.com/sulu/sulu/pull/5799

File: src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/UploadToolbarAction.js [hunk 2]
```diff
@@ -153,20 +258,20 @@ export default class UploadToolbarAction extends AbstractListToolbarAction {
     @computed get requestParameters(): $ReadOnly<Object> {
         const {
             options: {
-                requestParameters: attributesToRequest = {},
-                routerAttributesToRequest = {},
+                request_parameters: attributesToRequest = {},
+                router_attrbites_to_request: routerAttributesToRequest = {},
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/UploadToolbarAction.js:262]: Looks like a typoe: 

```suggestion
                router_attributes_to_request: routerAttributesToRequest = {},
```

File: src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/UploadToolbarAction.js [hunk 3]
```diff
@@ -153,20 +258,20 @@ export default class UploadToolbarAction extends AbstractListToolbarAction {
...
         } = this;
 
         if (!attributesToRequest || typeof attributesToRequest !== 'object') {
-            throw new Error('The "attributesToRequest" option must be an object!');
+            throw new Error('The "request_parameters" option must be an object!');
         }
 
         if (!routerAttributesToRequest || typeof routerAttributesToRequest !== 'object') {
-            throw new Error('The "routerAttributesToRequest" option must be an object!');
+            throw new Error('The "router_attrbites_to_request" option must be an object!');
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/UploadToolbarAction.js:274]: Same as above:

```suggestion
            throw new Error('The "router_attributes_to_request" option must be an object!');
```

**Review by alexander-schranz — CHANGES_REQUESTED** [review]

**Review by alexander-schranz — APPROVED** [review]

**Comment by alexander-schranz** [comment]
Is okay from my side @luca-rath do you want to have also a look at this?

**Review by luca-rath — APPROVED** [review]

**Pull request merged**